repository/repository.go

```diff
@@ -1,19 +1,29 @@
 package repository
 
 import (
+	"context"
 	"github.com/murilogilfelpeto/ps-tag-onboarding-go/configuration"
+	"github.com/murilogilfelpeto/ps-tag-onboarding-go/service/models"
 	"go.mongodb.org/mongo-driver/mongo"
 )
 
-var (
-	logger         *configuration.Logger
-	userCollection *mongo.Collection
-	context        = configuration.GetContext()
-)
+var logger = configuration.NewLogger("repository")
+
+type Repository interface {
+	Save(ctx context.Context, user models.User) (models.User, error)
+	GetUserById(ctx context.Context, id string) (models.User, error)
+	GetUserByFullName(ctx context.Context, firstName string, lastName string) (models.User, error)
+}
+
+type repository struct {
+	collection *mongo.Collection
+}
```

Comment:
Same comment about interfaces and the naming convention. Try to keep it consistent with golang standards.